sklearn/utils/setup.py

```diff
@@ -83,6 +83,11 @@ def configuration(parent_package='', top_path=None):
                          include_dirs=[numpy.get_include()],
                          libraries=libraries)
 
+    config.add_extension("_clibs_helpers",
```

Comment:
Maybe we should rename this to `_threaded_libs_helpers` because the goal is to control their thread pools. We don't really care that they are written in C or some other programming language.